### PR TITLE
init syntax, getting started page. 

### PR DESCRIPTION
**Pull request by Ydomat**
### Summary
Here goes a short summary about what the PR is about.

#### PR Type

- [ ] This PR is a code change that implements a feature request.
- [ ] This PR fixes an issue.
- [ ] This PR adds a new feature that is not an open feature request.
- [ X] This PR is not a code change (e.g. documentation, README, ...)
#### Other
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] If code changes were made then they have been tested.
- [ ] I have updated the documentation to reflect the changes.

### CODE REVIEW

File: docs/cheatsheets/pc_combat.rst [hunk 1]
```diff
@@ -41,39 +41,39 @@ Your turn
...
 To modify an attribute of a combatant (most common uses are -ac <AC>, -resist/immune/vulv <damage type>, -h)
 
-   !init opt <combatant name> <arguments>
+   !i opt <combatant name> <arguments>
 
    Most common Arguments:
      -ac <AC> (sets AC to new value)
      -resist/immune/vulv <damage type> (gives resistance, immunity, or vulnerability or specified type)
-     -h (toggles wether combatants AC and HP are hidden.)
+     -h (toggles weather combatants AC and HP are hidden.)
```
Comment (mommothazaz123) [inline docs/cheatsheets/pc_combat.rst]: whether, probably 😛

File: docs/cheatsheets/dm_combat.rst [hunk 2]
```diff
@@ -14,30 +14,30 @@ Starting combat
 
 First, combat must be started in a channel.  All commands should be posted in the **Same** discord channel that combat was started in.  Combat cant be started by using the following command:
```
Comment (mommothazaz123) [inline docs/cheatsheets/dm_combat.rst]: can't?

**Comment by Ydomat** [comment]
Yep. I’ll clean it up and add code blocks. I was looking for something else to set the commands apart but it seems code blocks might be the best option.

**Comment by Ydomat** [comment]
Also, I’ll add another wave of editing so random syntax and verbiage stuff doesn’t come through the PR’s.

**Comment by Ydomat** [comment]
Updates added.

**Pull request merged**